### PR TITLE
Query multiple tables

### PR DESCRIPTION
**Pull request by KanzaTahreem**
For the third milestone of the _Vet Clinic_ database project, I
- Created a table named `owners` with columns `id`, `full_name`, and `age`
- Created a table named `species` with columns `id` and `name`
- Modified the `animals` table
   - Removed column species
   - Added column `species_id` which is a foreign key referencing the `species` table 
   - Added column `owner_id` which is a foreign key referencing the `owners` table
- Inserted the given data into the `owners` table
- Inserted the given data into the `species` table
- Altered the `animals` table with the given data
- Modified the inserted animals to include values of `species_id` and  `owners_id`
- Performed queries using `JOIN`

### CODE REVIEW

File: queries.sql [hunk 1]
```diff
@@ -45,3 +45,19 @@ SELECT AVG(weight_kg) FROM animals;
 SELECT neutered, MAX(escape_attempts) FROM animals GROUP BY neutered;
 SELECT species, MIN(weight_kg), MAX(weight_kg) FROM animals GROUP BY species;
 SELECT species, AVG(escape_attempts) from animals WHERE date_of_birth BETWEEN '1990-01-01' and '2000-01-01' GROUP BY species;
+
+SELECT name FROM animals JOIN owners ON owners.full_name = 'Melody Pond' AND owners.id = animals.owner_id;
+SELECT animals.name FROM animals JOIN species ON species.name = 'Pokemon' AND animals.species_id = species.id;
+SELECT owners.full_name, animals.name FROM owners LEFT JOIN animals ON owners.id=animals.owner_id ORDER BY full_name;
+SELECT species.name, COUNT(animals.name) FROM animals JOIN species ON animals.species_id = species.id GROUP BY species.id;
```
Comment (Gambit142) [inline queries.sql:52]: - [ ] Great work writing the command that gives the accurate result. Your command is accurate but the screenshot you attached displays the wrong result for this query. Kindly update your screenshot to reflect the accurate result.

### Result of Query

image

### Screenshot Attached

image

#### Tip: There are `5 Pokemon` and `6 Digimon`. 👍 💻

**Review by Gambit142 — CHANGES_REQUESTED** [review]
Hi @KanzaTahreem 👋, 

Good job so far!

## Highlights
- All codes for creating, structuring and altering tables are in the _**schema.sql**_ file ✔️ 
- All codes that populate tables with data are in the _**data.sql**_ file ✔️ 
- All codes that query or help to analyze data are in the _**queries.sql**_ file ✔️ 

There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Kindly check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:

N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

File: queries.sql (hunk 1 above)
Comment (KanzaTahreem) [reply queries.sql:52]: Hello @Gambit142
Thank you for your feedback. I checked it twice and got the following result for `Pokemon` and `Digimon`. Please consider the following result. It shows that there are `4` rows of `Pokemon` and `6` rows of `Digimon`.
**Kindly advise me on this 😄**

image

Comment (KanzaTahreem) [reply queries.sql:52]: Hello @Gambit142 
I checked it again. In Tuesday's project, there was a query in the requirement to delete one animal. Before that, there were a total of `11` animals. But after the deletion, the number updated to `10`. I am attaching the screenshots of Tuesday's project requirement as well as my solution (which is already approved)

image
image
image

**So, the total number of animals present at a moment is `10`. In this case, `4` rows of Pokemon and `6` rows of Digimon make sense.**

**Kindly advise me on this. Waiting for your reply.**

**Review by uwadonat — APPROVED** [review]
## Hi @KanzaTahreem,

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
### Congratulations! 🎉 
### Highlights
- Good pull request message ✔️
- Professional redme file ✔️ 
approval1

# Status: Approved ✔️✔️✔️

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100%.
Please, remember to tag me @uwadonat in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

**Pull request merged**